atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -199,7 +193,7 @@ public void shouldNotAttemptMoreThanOneConnectionOnSuccess() {
 
     @Test
     public void shouldRetryOnSameNodeToFailureAndThenRedirect() {
-        int numHosts = CassandraClientPoolImpl.MAX_TRIES_TOTAL - CassandraClientPoolImpl.MAX_TRIES_SAME_HOST + 1;
+        int numHosts = 4;
```

Comment:
would be nice to make this `4` less magic; perhaps we can do this once we have config for max tries total/same host (if that's even something we want). Alternatively, we can use getters like we do later on in this test method.

Reply:
Yep, let's make this change when we have these parameters in the config. Added a TODO.